activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -0,0 +1,16 @@
+---
+type: Resource
+---
+
+# Criteria  for codebase stewardship
+
+Here are the criteria the staff of the Foundation for Public Code will use to help evaluate whether a codebase is ready for/conducive to incubation. They are used before and after the 'Assess' phase in the [codebase stewardship lifecycle](lifecycle.md).
```

Comment:
I don't understand the word conductive in this context, adds a lot of complexity, I would omit.

Reply:
```suggestion
Here are the criteria the staff of the Foundation for Public Code will use to help evaluate whether a codebase is ready and suitable for incubation. They are used before and after the 'Assess' phase in the [codebase stewardship lifecycle](lifecycle.md).
```